### PR TITLE
task.shutdown() added to every examples

### PR DESCRIPTION
**Pull request by chetkhatri**
What were proposed in this Pull Request ?
* At examples, task were created but not shutdown called except 1 example, that has been added as a standard practice.

### CODE REVIEW

**Comment by chetkhatri** [comment]
@alexholdenmiller Can you please review it and merge

File: examples/display_model.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def main():
             if world.epoch_done():
                 print("EPOCH DONE")
                 break
+    world.shutdown()
```
Comment (alexholdenmiller) [inline examples/display_model.py]: same with this one

File: examples/display_data.py [hunk 2]
```diff
@@ -37,6 +37,6 @@ def main():
             if world.epoch_done():
                 print("EPOCH DONE")
                 break
-
+    world.shutdown()
```
Comment (alexholdenmiller) [inline examples/display_data.py]: the `with` statement calls this automatically, so we can remove this one here

Comment (chetkhatri) [inline examples/display_data.py]: @alexholdenmiller  Okey.

File: examples/display_model.py (hunk 1 above)
Comment (chetkhatri) [inline examples/display_model.py]: @alexholdenmiller closing this

**Pull request closed without merging**